syz-ci/manager.go

```diff
@@ -855,7 +861,50 @@ func (mgr *Manager) uploadCoverReport() error {
 	return nil
 }
 
+func (mgr *Manager) uploadCoverStat(fuzzingMinutes int) error {
+	if !mgr.managercfg.Cover || mgr.cfg.CoverPipelinePath == "" {
+		return nil
+	}
+	resp, err := mgr.httpGET("/cover?json=1")
+	if err != nil {
+		return fmt.Errorf("failed to get /cover?json=1 report: %w", err)
+	}
+	defer resp.Body.Close()
+	jsonCover, err := io.ReadAll(resp.Body)
+	if err != nil {
+		return fmt.Errorf("failed to read /cover?json=1 body: %w", err)
+	}
+	curTime := time.Now().Format(time.RFC3339Nano)
+	ciCover, err := cover.ManagerToCI(jsonCover, cover.CIDetails{
+		Timestamp:      curTime,
+		FuzzingMinutes: fuzzingMinutes,
+		Arch:           mgr.lastBuild.Arch,
```

Comment:
BuildID may be good to have to match against database data and uploaded vmlinux file.

Reply:
Added it.